remote-api/src/main/java/org/alfresco/repo/webdav/WebDAVLockService.java

```diff
@@ -56,7 +56,9 @@
     void lock(NodeRef nodeRef, String userName, int timeout);
 
     void lock(NodeRef nodeRef, LockInfo lockInfo);
-    

```

Comment:
Additional method with explicit timeout value passing to avoid passing it through date operations, added to express intent behind calling the method